docs/src/overview/structure.md

```diff
@@ -1,77 +1,88 @@
 ---
-title: "Structure"
+title: Structure
 weight: 1
+description: Learn about how your Platform.sh environments are structured and which files control that structure.
 ---
 
-Every application you deploy on Platform.sh is built as a **virtual cluster**,
-containing a set of containers.
-The default branch of your Git repository is always deployed as the production cluster.
-Any other branch can be deployed as a development cluster.
+{{< note >}}
 
-By default, you can have up to three live development clusters at once,
-but you can buy more on a per-project basis.
+This page describes how things work on Grid projects.
```

Comment:
:bulb:
Would it be the right place to add some information regarding resource usage? This might help for the high-level understanding of the relationship between these components and how they make out the project.
For instance, how it's split across containers, how performance can be monitored.

Reply:
I think that kind of overview would be very helpful. I had thought of this page as more about the files and structure of the project itself, which doesn't really get into resources and how environments are structured. The diagram does that a little, but not as much in that direction. It's mostly just to understand the purpose of the parts and their relationships, rather than how they really work together. Might be worth opening an issue to get a new doc about that.